public/docs/_examples/dependency-injection/dart/lib/app_component.dart

```diff
@@ -17,23 +16,31 @@ import 'providers_component.dart';
 @Component(
     selector: 'my-app',
     template: '''
-    <h1>{{title}}</h1>
-    <my-car></my-car>
-    <my-injectors></my-injectors>
-    <my-tests></my-tests>
-    <h2>User</h2>
-    <p id="user">
-      {{userInfo}}
-      <button (click)=\'nextUser()\'>Next User</button>
-    <p>
-    <my-heroes id="authorized" *ngIf="isAuthorized"></my-heroes>
-    <my-heroes id="unauthorized" *ngIf="!isAuthorized"></my-heroes>
-  ''',
-    directives: const [CarComponent, HeroesComponent, InjectorComponent, ProvidersComponent],
+      <h1>{{title}}</h1>
+      <my-car></my-car>
+      <my-injectors></my-injectors>
+      <my-tests></my-tests>
+      <h2>User</h2>
+      <p id="user">
+        {{userInfo}}
+        <button (click)=\'nextUser()\'>Next User</button>
```

Comment:
The `\` seems redundant. Why single quote while elsewhere in the template double quotes are used?